projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -302,6 +355,21 @@ export class NodesApi {
     }
   }
 
+  async createLinkToFileName(originalFileName: string, originalFileParentId: string, destinationParentId?: string): Promise<NodeEntry> {
+    if (!destinationParentId) {
```

Comment:
I think it could be shortened to `destinationParentId = destinationParentId ?? originalFileParentId;`

Reply:
Change provided in [e0e334d](https://github.com/Alfresco/alfresco-content-app/pull/3509/commits/e0e334d2cfde0e61ee3301d2a82bf0312194ee00)